README.md

```diff
@@ -66,11 +66,9 @@ $ sidekick --health-path=/minio/health/ready --address :8000 http://minio{1...4}
 
 ### Two sites with 4 servers each
 ```
-$ sidekick --health-path=/minio/health/ready http://site1-minio{1...4}:9000 http://site2-minio{1...4}:9000
+$ sidekick --health-path=/minio/health/ready http://site1-minio{1...4}:9000,http://site2-minio{1...4}:9000
```

Comment:
Two command args are two sites @karlpokus a single command with `,` is the same site

```
     $ sidekick --health-path=/minio/health/cluster http://site1-minio{1...4}:9000,http://site1-minio{5...8}:9000 \
               http://site2-minio{1...4}:9000,http://site2-minio{5...8}:9000
```

Reply:
aha! Oh now I see. Maybe the wording could be better. Let me see if I can make it clearer. If I don't - we'll just close this pr.